src/main/resources/application.yml

```diff
@@ -85,7 +85,11 @@ staff-notifications:
 
 rpa-notifications:
   sender:
-  recipient:
+  sealedClaimRecipient: "${RPA_NOTIFICATIONS_RECIPIENT_SEALED_CLAIM:}"
```

Comment:
don't remap env variables, use the spring convention
I think it will map to `RPA_NOTIFICATIONS_SEALEDCLAIMRECIPIENT` but test it out and perhaps rename to get a better property name as its now mixed `-` and `camelCase`

Reply:
Mapped it to RPANOTIFICATIONS_SEALEDCLAIMRECIPIENT as per spring boot relaxed binding.